samples/subsys/logging/kernel_event_logger/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _kerneleventlogger_sample:
+
+Kerneleventlogger Sample
```

Comment:
How about:

     Kernel Event Logger Sample
     ##########################

Note the #### underline must be at least as long as the title above it.

Reply:
done